esp/src/src-react/hooks/grid.tsx

```diff
@@ -139,6 +139,7 @@ function useFluentStoreGrid({
 }: useFluentStoreGridProps): useFluentStoreGridResponse {
 
     const constColumns = useConst({ ...columns });
+    // const memoizedColumns = React.useMemo(() => { return { ...columns } }, [columns]);
```

Comment:
@GordonSmith This is the change that was causing the odd click behaviour, where the grid row was capturing the click first, requiring a second click on the anchor in the now selected row to actually navigate to a new url.